packages/salesforcedx-vscode-lwc/src/service/devServerService.ts

```diff
@@ -59,11 +61,8 @@ export class DevServerService {
     return this.baseUrl;
   }
 
-  public setBaseUrl(url: string) {
-    this.baseUrl = url;
-  }
-
   public setBaseUrlFromDevServerUpMessage(data: string) {
+    data = stripAnsi(data);
```

Comment:
generic code styles comment
```suggestion
    const sanitizedData = stripAnsi(data);
```

Reply:
https://eslint.org/docs/rules/no-param-reassign